src/App.tsx

```diff
@@ -15,37 +17,98 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum SortType {
+  sort_field_alphabet = 'alphabet',
+  sort_field_length = 'length',
+  sort_field_none = '',
+}
+
+function getPreparedGoods(
+  goods: string[],
+  sortField: SortType,
+  isReversed: boolean,
+): string[] {
+  const preparedGoods = [...goods];
+
+  switch (sortField) {
+    case SortType.sort_field_alphabet:
+      preparedGoods.sort((good1: string, good2: string) =>
+        good1.localeCompare(good2),
+      );
+      break;
+
+    case SortType.sort_field_length:
+      preparedGoods.sort(
+        (good1: string, good2: string) => good1.length - good2.length,
+      );
+      break;
+
+    default:
+      break;
+  }
+
+  if (isReversed) {
+    preparedGoods.reverse();
+  }
+
+  return preparedGoods;
```

Comment:
[CODE KNOWLEDGE] - The function name `getPreparedGoods` doesn't start with a verb indicating what it does. Consider renaming it to something like `prepareGoods` to follow best practices for function naming.